scripts/togglePasswordVisibility.js

```diff
@@ -0,0 +1,10 @@
+// 비밀번호 보이기/숨기기 토글 함수
+export function togglePasswordVisibility(passwordInput, toggleIcon) {
+  const type =
+    passwordInput.getAttribute("type") === "password" ? "text" : "password";
+  passwordInput.setAttribute("type", type);
+  toggleIcon.src =
+    type === "password"
+      ? "../image/icons/eye-off.svg"
+      : "../image/icons/eye-on.svg";
```

Comment:
위의 타입처럼 image url도 변수로 빼주시면 가독성에 더 좋을 것 같습니다.

```suggestion
  const imageUrl =  type === "password" ? "../image/icons/eye-off.svg" : "../image/icons/eye-on.svg";
  toggleIcon.src = imageUrl;
```

Reply:
반영했습니다.